sequencer/src/main/java/net/consensys/linea/extradata/LineaExtraDataPlugin.java

```diff
@@ -75,10 +75,11 @@ public void start() {
         extraDataHandler.handle(initialExtraData);
       } catch (final Exception e) {
         log.warn(
-            "Failed setting initial pricing conf from extra data field ({}) of the chain head block {}({})",
+            "Failed setting initial pricing conf from extra data field ({}) of the chain head block {}({}), error {}",
```

Comment:
Can we log the whole stacktrace on a lower log level?

Reply:
is it ok also if the stack trace is logged in this warn statement, without adding a new one?

Reply:
I'm not sure about that TBH, up to you. But in the vast majority of cases just the error message should be sufficient.

Reply:
the new version logs the error message and the stack trace